src/main/java/tech/jhipster/lite/project/infrastructure/secondary/FileSystemProjectsRepository.java

```diff
@@ -84,4 +90,13 @@ public ProjectHistory getHistory(ProjectPath path) {
   private Path historyFilePath(ProjectPath path) {
     return Paths.get(path.get(), HISTORY_FOLDER, HISTORY_FILE);
   }
+
+  @Override
+  public Collection<Preset> getPreset() {
```

Comment:
Since this is a collection, maybe:
```suggestion
  public Collection<Preset> getPresets() {
```